src/pages/App.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
@hammeiam I wasn't able to figure out how to resolve these TS errors for the loadable library which is necessary for code splitting. Do you happen to know how to resolve it for lines 35 & 36?

Reply:
I think we can just use React.lazy since we're not SSR